### DISCUSSION TITLE
Getting Start with Google Summer of Code (GSOC)

### DISCUSSION THREAD

**Discussion by ethanwhite** [weecology/DeepForest#628] [Announcements]
Welcome to everyone engaging with the project as part of exploring/applying for Google Summer of Code. The best way to get started is:

1. Follow the [Developers Guide](https://deepforest.readthedocs.io/en/latest/CONTRIBUTING.html) to install the package and get the tests up and running locally.
2. Take a look [issues labeled Good First Issue](https://github.com/weecology/DeepForest/labels/good%20first%20issue). Find one you'd like to try working on that someone else isn't already working on. Then start by replicating the issue locally and leave a note in the issue that you are working on it.
3. If you can solve the issue submit a Pull Request with your solution. If not, tell us what you did and where you got stuck in the issue and we'll do our best to help you out.
4. Once you've got a feel for the software by doing the above you can open [a new Discussion](https://github.com/weecology/DeepForest/discussions/new?category=general)  to discussion to ask questions about your project ideas.

**Comment by Abhishek-kumar0503** [comment]
Hello @ethanwhite,  
I have successfully setup deepforest repo in my local device, and it's running fine.I am interested in the project  Advancing Bird Detection and Classification in Hand-Held Airborne Imagery. Can you please guide me further on this project?  Also i solve 2-3 good first issue.Please review my Pull Request.
